core/templates/components/summary-tile/collection-summary-tile.component.html

```diff
@@ -0,0 +1,61 @@
+<oppia-learner-dashboard-icons *ngIf="showLearnerDashboardIconsIfPossible && userIsLoggedIn"
+                               [activityType]="activityTypeCollection"
+                               [activityId]="getCollectionId"
+                               [activityTitle]="getCollectionTitle"
+                               [isContainerNarrow]="isContainerNarrow"
+                               [activityActive]="collectionIsCurrentlyHoveredOver"
+                               [isAddToPlaylistIconShown]="!isOwnedByCurrentUser">
+</oppia-learner-dashboard-icons>
+<mat-card class="oppia-activity-summary-tile protractor-test-collection-card"
+         [ngClass]="{'oppia-activity-playlist-tile': isPlaylistTile}" [style.padding]="0">
+  <a [href]="getCollectionLink()" class="thumbnail">
+    <div class="title-section" [ngStyle]="{'background-color': getThumbnailBgColor}">
+      <img class="collection-corner-image" [src]="getStaticImageUrl('/general/collection_corner.svg')" alt="">
+      <img class="thumbnail-image" [src]="getCompleteThumbnailIconUrl()" alt="{{getCategory}}">
+      <h3 class="activity-title protractor-test-collection-summary-tile-title">{{getCollectionTitle || defaultEmptyTitle}}</h3>
+    </div>
+
+    <div [ngStyle]="isPlaylistTile && {'float': 'right', 'width': '67%', 'margin-top': '3%'}">
+      <div class="objective">
+        {{getObjective | truncateAndCapitalize: 95}}
+        <span *ngIf="!getObjective">No objective specified.</span>
+      </div>
+      <ul class="metrics layout-align-space-between-center layout-row">
+        <li>
+          <span>
+            COLLECTION ({{getNodeCount}})
```

Comment:
Can we add spaces after {{ and before }} (everywhere else too)

Reply:
done!